kustomize/base/manager/resources/deploy.yaml

```diff
@@ -18,6 +18,11 @@ spec:
         - image: event-bridge-manager
           imagePullPolicy: IfNotPresent
           name: event-bridge-manager
+          env:
+            - name: EVENT_BRIDGE_RESOURCES_WORKER_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
I'm no expert :smile: 

Can you explain how `metadata.name` will resolve to a **different** value for each container running the `manager`? Won't `metadata.name` always resolve to `event-bridge-manager` (i.e. the name of the Kubernetes resource for our `manager`).

Reply:
I had a hunt around and `metadata.name` may furnish us with a unique Pod name however for that to be useful for us we'd either need (1) a single container per Pod, (2) use something other than `metadata.name`. Our `WorkerId` is used to pick up a list of `Work` for processing. If we have the risk of the same value across multiple containers within a Pod then it's possible we'd have multiple instances of `WorkManagerImpl` processing the same work at the same time... try debugging that!

Reply:
I live and learn: https://kubernetes.io/docs/concepts/workloads/pods/#workload-resources-for-managing-pods

"**Pods that run a single container**. The "one-container-per-Pod" model is the most common Kubernetes use case; in this case, you can think of a Pod as a wrapper around a single container"